src/Swashbuckle.AspNetCore.Annotations/Swashbuckle.AspNetCore.Annotations.csproj

```diff
@@ -8,7 +8,7 @@
     <NoWarn>$(NoWarn);1591</NoWarn>
     <PackageTags>swagger;documentation;discovery;help;webapi;aspnet;aspnetcore;annotations</PackageTags>
     <SignAssembly>true</SignAssembly>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.0;net5.0;net6.0;net7.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net6.0;net8.0;net9.0</TargetFrameworks>
```

Comment:
From my experience, it's better to put the most recent TFM first. When cloning the repo for the first time and opening a file in Visual Studio, that's the version being activated in the code editor. So when newer APIs are available in .NET 9, IntelliSense will show them, so developers are aware. Of course, then they'll need to use `#if` to fix up older targets, but at least the new APIs are being shown.

Reply:
I'm unable to resolve this conversation (I don't know why), but it looks like this was addressed and can be closed.